game-engine/src/main/scala/za/co/entelect/challenge/game/contracts/map/Block.scala

```diff
@@ -4,15 +4,21 @@ import za.co.entelect.challenge.game.contracts.Config.Config
 
 class Block(position: BlockPosition, var mapObject: Int, var occupiedByPlayerWithId: Int) {
 
+    private var hasCyberTruck: Boolean = false;
+
     override def toString() : String = { 
         val stringRepresentation =
             if(occupiedByPlayerWithId != Config.EMPTY_PLAYER) occupiedByPlayerWithId.toString
+            else if(isOccupiedByCyberTruck()) "C"
             else if(mapObject == Config.EMPTY_MAP_OBJECT) "░"
             else if(mapObject == Config.MUD_MAP_OBJECT) "▓"
             else if (mapObject == Config.BOOST_MAP_OBJECT) "»"
             else if (mapObject == Config.FINISH_LINE_MAP_OBJECT) "║"
             else if (mapObject == Config.OIL_ITEM_MAP_OBJECT) "Φ"
             else if (mapObject == Config.OIL_SPILL_MAP_OBJECT) "█"
+            else if (mapObject == Config.WALL_MAP_OBJECT) "#"
+            else if (mapObject == Config.LIZARD_MAP_OBJECT) "&"
```

Comment:
Press    `win+.  ` This opens up that emoji window. Choose the symbols tab. These are all characters that show up in the console, like the oil barrel Φ icon
![image](https://user-images.githubusercontent.com/21290218/81713450-2d94ce00-9476-11ea-8ddf-99defaa86733.png)

```suggestion
            else if (mapObject == Config.LIZARD_MAP_OBJECT) "∱"
```
⨘  ⨙  ⨑

Reply:
Used the suggestion